contrib/test_jenkins.sh

```diff
@@ -187,7 +187,16 @@ build_release_pkg() {
 	# Show UCX info
 	./src/tools/info/ucx_info -f -c -v -y -d -b -p -w -e -uart
 
-	if [ -x /usr/bin/dpkg-buildpackage ]; then
+	set +e
+	out=$(rpm -q rpm 2>/dev/null)
+	rc=$?
+	set -e
+	rpm_based=yes
+	if [[ $rc != 0 || "$out" == *"not installed"* ]]; then
```

Comment:
maybe just do "which rpm" or [ -x /bin/rpm ] ?

Reply:
Package `rpm` can be installed on deb based system, so it will be present.

Reply:
https://ask.fedoraproject.org/en/question/49738/how-to-check-if-system-is-rpm-or-debian-based/

Reply:
maybe this will work:

```
if [ -f /etc/debian_version ]; then
    echo "This is debian based distro"
elif [ -f /etc/redhat-release ]; then
    echo "This is RedHat based distro"
else
    echo "This is something else"
fi
```

Reply:
@miked-mellanox saw the link, doesn't work for non-standard paths. Checking `/etc/` files is tricky, need to take into account many distros.